### PR TITLE
Fix of build errors

### PR DESCRIPTION
**Pull request by Khukhuna**
- Fixed build error (Missing comma in angular.json file)
- Removed html, head & body tag from certificate-template.component
- Also angular parser was getting confused & was throwing error, because of  ```{ }``` chars in _certificate-template.component.html_
- Removed duplicate import of ```font-awesome```
- Removed importing ```materialize``` from angular.json, becuase it was breaking login ui [img](https://user-images.githubusercontent.com/37747169/49590130-7ba1ca00-f984-11e8-8157-9d8d0a197497.png)

### CODE REVIEW

**Comment by sidhyatikku** [comment]
I was about to create an issue for the same and it looks like you re-did #116 😆 🎉

File: angular.json [hunk 1]
```diff
@@ -25,8 +25,6 @@
             "styles": [
               "node_modules/bootstrap/dist/css/bootstrap.min.css",
               "src/styles.css",
-              "https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css",
```
[the remark this answers is not in the document]
Comment (Khukhuna) [reply angular.json]: If we'll add materialize from node_modules, it'll change look in every component (We'll have to redesign every component).

Preview of login screen if we'll add materialize in angular.json:
_(input gets bigger, text & buttons get smaller)_

**Comment by prithaupadhyay** [comment]
Great @Khukhuna 🎉. Thank you for fixing those errors.

**Comment by aashutoshrathi** [comment]
I think this should be merged ASAP.

**Comment by prithaupadhyay** [comment]
@aashutoshrathi Okay. I'll just merge it after testing. Thanks.

**Pull request merged**